docs/adrs/0002-add-system-agent-support-to-wins.md

```diff
@@ -0,0 +1,19 @@
+# 2. Add System Agent support to Wins
+
+Date: 2021-10-04
+
+## Status
+
+Accepted
+
+## Context
+
+The [System Agent]() is used to install and upgrade RKE2 using plans. Support is needed to provide this ability for Windows. 
+
+## Decision
+
+System Agent functionality needs some capabilities of Wins to achieve its goal. This would require that both the system agent service and wins service to be running on a system. Alternatively, we could consume the system agent as a package in the wins service reducing the need to having both installed in a system.
```

Comment:
nit: Seems like this should be part of the Context, not the decision. The decision is just the last sentence right?